dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1240,11 +1305,43 @@ def _regenerate_histogram(
             new_bin_edges = np.linspace(
                 options["min_edge"], options["max_edge"], suggested_bin_count + 1
             )
+        return self._assimilate_histogram(
+            entity_count_by_bin,
+            bin_edges,
+            new_bin_counts,
+            new_bin_edges,
+            suggested_bin_count,
+        )
 
+    def _assimilate_histogram(
+        self,
+        entity_count_by_bin: np.ndarray,
+        bin_edges: np.ndarray,
+        new_entity_count_by_bin: np.ndarray,
+        new_bin_edges: np.ndarray,
```

Comment:
Trying to think how we can bring clarity to the word assimilate. instead of `new`, maybe `destination`